Mathlib/Topology/ContinuousFunction/FunctionalCalculus.lean

```diff
@@ -582,6 +582,45 @@ lemma eq_one_of_spectrum_subset_one (h_spec : spectrum R a ⊆ {1}) (ha : p a :=
     a = 1 := by
   simpa using eq_algebraMap_of_spectrum_subset_singleton a 1 h_spec
 
+lemma spectrum_algebraMap_subset (r : R) : spectrum R (algebraMap R A r) ⊆ {r} := by
+  rw [← cfc_const r 0 (cfc_predicate_zero R),
+    cfc_map_spectrum (fun _ ↦ r) 0 (cfc_predicate_zero R)]
+  rintro - ⟨x, -, rfl⟩
+  simp
+
+lemma spectrum_algebraMap_eq [Nontrivial A] (r : R) :
```

Comment:
Can you namespace the ones that don't have `cfc` to the `CFC` namespace? (including the few lemmas above that you didn't write)

It's strange to have `spectrum_zero_eq` for example, in the root namespace, with a `ContinuousFunctionalCalculus` hypothesis, especially when we have `spectrum.zero_eq` already.

Reply:
Yes, fair enough.